js/src/forum/components/DiscussionComposer.js

```diff
@@ -92,7 +92,9 @@ export default class DiscussionComposer extends ComposerBody {
     app.store.createRecord('discussions').save(data).then(
       discussion => {
         app.composer.hide();
-        app.cache.discussionList.addDiscussion(discussion);
+
+        if (app.cache.discussionList) app.cache.discussionList.addDiscussion(discussion);
```

Comment:
This is likely made obsolete by #1868.

Reply:
That PR would still need a check anyway... I'll remove the extra lines I added so maybe git just sees it as adding the `if` before, and not changing the whole line.